cordova-plugin-outline/android/aidl/org/outline/shadowsocks/ShadowsocksConfig.aidl

```diff
@@ -19,4 +19,5 @@ parcelable ShadowsocksConfig {
   int port;
   String password;
   String method;
+  @nullable byte[] prefix;
```

Comment:
@jyyi1 We'll probably be able to get rid of this if we share protocol buffers across implementations